sqlmesh/core/model/definition.py

```diff
@@ -461,7 +461,11 @@ def ctas_query(self, **render_kwarg: t.Any) -> exp.Query:
             )
         return query
 
-    def referenced_audits(self, audits: t.Dict[str, ModelAudit]) -> t.List[ModelAudit]:
+    def referenced_audits(
+        self,
+        audits: t.Dict[str, ModelAudit],
+        default_audits: t.Optional[t.List[AuditReference]] = None,
```

Comment:
Small nit: instead of making this default to `None`, is there any reason why it couldn't default to `[]` instead?

Then below where you have:
```
self.audits + default_audits if default_audits else self.audits
```

It could become:
```
self.audits + default_audits
```

Reply:
yes this makes sense changed it